cdap-api-spark-base/src/main/scala/io/cdap/cdap/api/spark/SparkExecutionContextBase.scala

```diff
@@ -215,4 +216,10 @@ trait SparkExecutionContextBase extends RuntimeContext
     * Returns a new instance of [[io.cdap.cdap.api.spark.JavaSparkExecutionContext]].
     */
   def toJavaSparkExecutionContext(): JavaSparkExecutionContext
+
+  /**
+   *
+   * @return a map of feature flag names and if they are enabled.
+   */
+  def getFeatureFlags(): util.Map[String, java.lang.Boolean]
```

Comment:
You shouldn't need to redefining the method here since it is coming from the `RuntimeContext`.